apisix/plugins/kafka-logger.lua

```diff
@@ -32,6 +32,11 @@ local buffers = {}
 local schema = {
     type = "object",
     properties = {
+        meta_style = {
```

Comment:
`meta_format` maybe a better name.

any suggestion?

Reply:
yes, `meta_format` is better